packages/manager/modules/nasha/src/dashboard/partition/partition.routing.js

```diff
@@ -65,27 +71,27 @@ export default /* @ngInject */ ($stateProvider) => {
           serviceName,
         }).$promise.then((partitions) => partitions.map(preparePartition));
       },
-      partitionApiUrl: /* @ngInject */ (serviceName, partitionName) =>
-        `/dedicated/nasha/${serviceName}/partition/${partitionName}`,
+      partitionApiUrl: /* @ngInject */ (nashaApiUrl, partitionName) =>
+        `${nashaApiUrl}/partition/${partitionName}`,
       partitionName: /* @ngInject */ ($transition$) =>
         $transition$.params().partitionName,
       partitionHref: /* @ngInject */ ($state, serviceName) => () =>
         $state.href(STATE_NAME, { serviceName }),
-      tasks: /* @ngInject */ (
-        iceberg,
-        serviceName,
-        partitionName,
-        NashaTask,
-        prepareTasks,
-      ) =>
-        iceberg(`/dedicated/nasha/${serviceName}/task`)
-          .query()
-          .expand('CachedObjectList-Pages')
-          .addFilter('status', 'in', Object.values(NashaTask.status))
-          .addFilter('partitionName', 'eq', partitionName)
-          .execute(null, true)
-          .$promise.then(({ data: tasks }) => prepareTasks(tasks)),
+      trackTasks: /* @ngInject */ ($state) => (params) =>
+        $state.go(TASK_TRACKER_STATE_NAME, params),
```

Comment:
**nitpick:** function name suggests it does something related to tracking while it does a state navigation. Can you rename according to its responsibility?